wordpress-mobile/WordPress-iOS
# Notifications > Comment Details: show previous and next arrows in nav bar

Ref: #17790 

On the new Notification Comment details view, previous and next buttons are now displayed in the nav bar. 

They're not hooked up yet, so:
- For now tapping them simply logs a message (just to show they work).
- They are not disabled if there is no previous/next notification.

To test:

- Enable `notificationCommentDetails`.
- Go to Notifications > Comments > notification details.
  - The Edit button is always displayed (all devices, all orientations).
  - On an iPad, the arrow buttons are never displayed.
  -  On a regular size iPhone, the arrow buttons are always displayed.
  - On an X size iPhone:
    - In portrait, the arrow buttons are displayed.
    - In landscape, the arrow buttons are not displayed.  
- Go to My Site > Comments > comment details.
  - Verify only the Edit button is displayed.

| Arrows | No Arrows |
|--------|-------|
| ![arrows_shown](https://user-images.githubusercontent.com/1816888/152617827-daef69d4-56a5-402a-ac80-05fd7ee4ec88.png) | ![arrows_not_shown](https://user-images.githubusercontent.com/1816888/152617824-a2991a1f-2f33-419e-98a6-01dc24fd5c1f.png) |


## Regression Notes
1. Potential unintended areas of impact
N/A. Feature disabled.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
